src/histogram_2d.ts

```diff
@@ -143,34 +145,38 @@ class Histogram2D implements IAppView {
       .then((json) => {
         const data = [];
```

Comment:
1. Store the raw data in a class variable
2. Do the calcuation (below) based on the active change types
3. Then show the (filtered) data (https://github.com/Caleydo/taco/pull/89/files#diff-92a6173c7bb6c10348cac572104b3722R135)